packages/combine/combine.0.6/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+name: "combine"
+version: "0.6"
+License: "GNU Library General Public License version 2.1"
```

Comment:
Also it's better to use `spdx` machine readable identifiers (this may be linted by opam in the future). In this case use [`LGPL-2.1](https://spdx.org/licenses/LGPL-2.1.html)